net.shrew.ike.qikea.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "net.shrew.ike.qikea",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "command": "qikea",
+    "sdk": "org.freedesktop.Sdk",
+    "finish-args": [
+        "--filesystem=host",
+        "--filesystem=/run/ikedi",
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11"
+    ],
+    "modules": [
```

Comment:
You should add `cleanup` array (global or per module) to remove static libraries and development headers.

Reply:
Ok, I will review if any artifact is present after building

Reply:
Done with 92cbc7c